docker/install/ubuntu_install_caffe.sh

```diff
@@ -22,13 +22,40 @@ set -o pipefail
 
 apt-get update --fix-missing
 
-# The precompiled caffe dependents on tzdata.
-# While installing tzdata in docker, we need set the time zone manually,
-# which will cause the container to hang during installation.
-# So in order to avoid manually selecting the time zone, set as following:
-export DEBIAN_FRONTEND=noninteractive
-apt-get install -y tzdata
+# Install dependencies
+apt-get install -y --no-install-recommends libboost-filesystem-dev libboost-python-dev \
+    libboost-system-dev libboost-thread-dev libboost-regex-dev protobuf-compiler \
+    libprotobuf-dev libhdf5-serial-dev libopenblas-dev libgflags-dev libgoogle-glog-dev
+rm -rf /var/lib/apt/lists/*
 
-apt-get install caffe-cpu -y
+# install python packages
+pip install "numpy>=1.7.1"
```

Comment:
can you add these to `python/gen_requirements.py`? also, can you use just one pip command to do this?

Reply:
Ok.